pkg/utils/docker.go

```diff
@@ -0,0 +1,40 @@
+package utils
+
+import (
+	"fmt"
+
+	"github.com/Sirupsen/logrus"
+
+	"github.com/kubernetes-incubator/kompose/pkg/kobject"
+)
+
+func DockerBuildImage(name string, service kobject.ServiceConfig, composeFileDir string) string {
+	image := name
+	if service.Image != "" {
+		image = service.Image
+	}
+
+	cmd := NewCommand(fmt.Sprintf("docker build -t %s %s", image, service.Build))
```

Comment:
@kadel as you can see here: https://github.com/docker/docker/blob/master/cli/command/image/push.go#L32, consuming the docker's cli lib is not that simple. You need to explicitly set registry auth params. Whereas the docker CLI takes care of it, implicitly. I am not sure if we should proceed with using the go library.